.github/workflows/main.yml

```diff
@@ -118,10 +120,22 @@ jobs:
       # Development and Feature branches
       # --------------------------------------------
       - name: Build - Assemble
-        uses: virtualsatellite/ci-actions/ci-maven-simple-build-action@v4
+        uses: virtualsatellite/ci-actions/ci-maven-simple-build-action@v5-beta
```

Comment:
V5-Beta in the release doesn't look that nice to me. Maybe we should create a v5 out of the beta if we ware happy with it.

Reply:
Updated to v5 release.